### PR TITLE
Added link to instructions on using SymbolSource to readme

### PR DESCRIPTION
**Pull request by gb92**
Added link to information on enabling SymbolSource for debugging to Useful Resources section of the Readme

### CODE REVIEW

**Comment by gb92** [comment]
@phvannor, @fabiocav can you review the wording for adding info about SymbolSource to the readme?

**Comment by fabiocav** [comment]
The URL NuGet points to is this: http://www.symbolsource.org/Public/Home/VisualStudio
It's pretty much the same content, but I'd probably use that instead.

Aside from that, it looks good.

**Comment by fabiocav** [comment]
@gb92 , ping

File: README.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ Windows Phone 8 and .NET 4.5.
 * [E2E Test Suite](https://github.com/Azure/azure-mobile-services-test)
```
Comment (pragnagopa) [inline README.md]: Delete [E2E Test Suite] link

**Pull request closed without merging**